.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -464,11 +464,9 @@ def write(x=''):
     
     for file in [WORKERS_JSON, CONTAINERS_JSON]:
       shutil.chown(file, 'root',DAEMON_GID)
-      os.chmod(file, 0o460)
+      os.chmod(CONTAINERS_JSON, 0o460)
```

Comment:
I think this might be a typo: you're only chmodding the containers json file in this loop. I think what you want is for it to chmod ```file``` and then after the loop to chgrp ```CONTAINERS_JSON``` to ```daemon_php```